examples/neighbors/plot_lmnn_classification.py

```diff
@@ -0,0 +1,54 @@
+"""
+============================================
+Large Margin Nearest Neighbor Classification
+============================================
+
+Sample usage of Large Margin Nearest Neighbors classification.
+It will plot the decision boundaries for each class.
+"""
+print(__doc__)
```

Comment:
Hi, thanks for the comments! I added author and licence comments.
But are you sure about the move? All files I see in the examples directory have it like this.